lib/cucumber/cli/rerun_file.rb

```diff
@@ -10,7 +10,7 @@ def self.can_read?(path)
       end
 
       def self.real_path(path)
-        path[1..-1] # remove leading @
+        path[1..] # remove leading @
```

Comment:
Can we amend this comment above this, and be a bit more verbose (If I'm correct), saying.

`# This removes the leading @ from path`

It looks like maybe we need to fix something? Not sure.... Can we check

Reply:
I've just removed the comment for now